diff --git a/lib/index.cjs b/lib/index.cjs
index 1649e447b29cd9df3717eaace7dd010e982e2114..ab49459392a3b5c12c2ead5bad534d312503432c 100644
--- a/lib/index.cjs
+++ b/lib/index.cjs
@@ -251,9 +251,8 @@ const cjsRequire = importMetaUrl$1 ? Module.createRequire(importMetaUrl$1) : (
   require
 );
 
-const { name, version } = cjsRequire(
-  "../package.json"
-);
+const name = "eslint-plugin-import-x";
+const version = "4.11.0";
 const meta = { name, version };
 
 var __defProp$e = Object.defineProperty;
diff --git a/lib/meta.js b/lib/meta.js
index 079bdb09192305267ea1b45b835f5138ffef3d19..1e8788be4bbf98e20f8ffcf2c44e993deea1b9f0 100644
--- a/lib/meta.js
+++ b/lib/meta.js
@@ -1,4 +1,4 @@
-import { cjsRequire } from './require.js';
-export const { name, version } = cjsRequire('../package.json');
+export const name = "eslint-plugin-import-x";
+export const version = "4.11.0";
 export const meta = { name, version };
 //# sourceMappingURL=meta.js.map
