diff --git a/lib/index.cjs b/lib/index.cjs
index 3a4efbb093a55d93eb35c954822ee8c618f632ca..483f69efc399cea3d8ea28e2401f9f5e9fee1fc0 100644
--- a/lib/index.cjs
+++ b/lib/index.cjs
@@ -279,7 +279,8 @@ const cjsRequire = importMetaUrl$1 ? (0, node_module.createRequire)(importMetaUr
 
 //#endregion
 //#region src/meta.ts
-const { name, version } = cjsRequire("../package.json");
+const name = "eslint-plugin-import-x";
+const version = "4.16.1";
 const meta = {
 	name,
 	version
diff --git a/lib/meta.js b/lib/meta.js
index 079bdb09192305267ea1b45b835f5138ffef3d19..51d7480496eaaadc1f19f94251bdc35afd666bed 100644
--- a/lib/meta.js
+++ b/lib/meta.js
@@ -1,4 +1,5 @@
 import { cjsRequire } from './require.js';
-export const { name, version } = cjsRequire('../package.json');
+export const name = "eslint-plugin-import-x";
+export const version = "4.16.1";
 export const meta = { name, version };
 //# sourceMappingURL=meta.js.map
