diff --git a/lib/index.js b/lib/index.js
index 834e5f28b07079df432e14b12281fef5fefda04f..5281310e7ced250ac1445015e2b9bd10ec9422f3 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -7,32 +7,6 @@
 
 const plugin = {
   meta: require('./meta'),
-  configs: {
-    // eslintrc configs
-    base: require('./configs/base'),
-
-    'vue2-essential': require('./configs/vue2-essential'),
-    'vue2-strongly-recommended': require('./configs/vue2-strongly-recommended'),
-    'vue2-recommended': require('./configs/vue2-recommended'),
-
-    essential: require('./configs/vue3-essential'),
-    'strongly-recommended': require('./configs/vue3-strongly-recommended'),
-    recommended: require('./configs/vue3-recommended'),
-
-    // flat configs
-    'flat/base': require('./configs/flat/base.js'),
-
-    'flat/vue2-essential': require('./configs/flat/vue2-essential.js'),
-    'flat/vue2-strongly-recommended': require('./configs/flat/vue2-strongly-recommended.js'),
-    'flat/vue2-recommended': require('./configs/flat/vue2-recommended.js'),
-
-    'flat/essential': require('./configs/flat/vue3-essential.js'),
-    'flat/strongly-recommended': require('./configs/flat/vue3-strongly-recommended.js'),
-    'flat/recommended': require('./configs/flat/vue3-recommended.js'),
-
-    // config-format-agnostic configs
-    'no-layout-rules': require('./configs/no-layout-rules')
-  },
   rules: {
     'array-bracket-newline': require('./rules/array-bracket-newline'),
     'array-bracket-spacing': require('./rules/array-bracket-spacing'),
